runtime/near-vm-logic/src/logic.rs

```diff
@@ -2554,8 +2554,8 @@ impl std::fmt::Debug for VMOutcome {
         };
         write!(
             f,
-            "VMOutcome: balance {} storage_usage {} return data {} burnt gas {} used gas {}",
-            self.balance, self.storage_usage, return_data_str, self.burnt_gas, self.used_gas
+            "VMOutcome: balance {} storage_usage {} return data {} burnt gas {} used gas {} logs {:?}",
```

Comment:
How likely is it that `logs` will be shown with many lines in the output? If it is “quite likely”, then I think it may be quite important that we figure out how to pass through the `#` formatting flag here. This way the callers of this `Debug` implementation can get a choice of printing 1 log line per line of output (i.e. pretty printed).

You can check for the flag by calling [`Formatter::alternate`](https://doc.rust-lang.org/stable/std/fmt/struct.Formatter.html#method.alternate)